### ISSUE TITLE
Support for docker's volume read only / read write options

### ISSUE THREAD

**Issue by adischik** [Tapad/sbt-docker-compose#60]
In docker compose files from version 2 there is an option to state if the volume is read only or read-write enabled.

For example: 
    
    version: '2'

    services:

      rabbit:

        image: rabbitmq:management

        ports:

         - "5672:5672"

         - "15672:15672"

        volumes:

         - ~/data/event_distribution/rabbit:/var/lib/rabbitmq:rw

         - ./env/rabbit/definitions.json:/etc/rabbitmq/definitions.json:ro

         - ./env/rabbit/rabbitmq.config:/etc/rabbitmq/rabbitmq.config:ro

note the `:rw` and `:ro` at the end of each volume definition

I would love for this to be supported. Thanks!

**Comment by kurtkopchik** [comment]
Hi @adischik - Thank you for reporting this issue!

I've made an update to the volume parsing code to handle the additional access specifier that can exist at the end of a volume definition starting with Docker Compose 2.0.

I included the change in the `1.0.22-SNAPSHOT` release. Let me know if it works for you and I'll get it merged into master and included in the next official release.

```
resolvers += "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/"
addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.22-SNAPSHOT")
```
